microbench/src/test/java/io/netty/microbench/util/AbstractMicrobenchmarkBase.java

```diff
@@ -42,7 +42,7 @@
     protected static final int DEFAULT_MEASURE_ITERATIONS = 10;
     protected static final String[] BASE_JVM_ARGS = {
         "-server", "-dsa", "-da", "-ea:io.netty...", "-XX:+AggressiveOpts", "-XX:+UseBiasedLocking",
-        "-XX:+UseFastAccessorMethods", "-XX:+UseStringCache", "-XX:+OptimizeStringConcat",
```

Comment:
`"-XX:+UseStringCache"` is not supported on java 8 and was throwing errors.  I could conditionally include it based upon java version....but I just removed to see what people thought.